go/vt/vtgate/plugin_mysql_server.go

```diff
@@ -65,7 +65,9 @@ func (vh *vtgateHandler) ConnectionClosed(c *mysqlconn.Conn) {
 	// Rollback if there is an ongoing transaction. Ignore error.
 	ctx := context.Background()
 	session, _ := c.ClientData.(*vtgatepb.Session)
-	_, _, _ = vh.vtg.Execute(ctx, session, "rollback", make(map[string]interface{}))
+	if (session != nil) {
```

Comment:
Please no surrounding parentheses in Go :)

The gofmt Git commit hook would have caught this :)

I sent https://github.com/youtube/vitess/pull/2836 out to fix this.